packages/villus/src/useQuery.ts

```diff
@@ -24,10 +24,10 @@ export interface QueryCompositeOptions<TData, TVars> {
   skip?: QueryPredicateOrSignal<TVars>;
 }
 
-interface QueryExecutionOpts<TVars> {
+export type QueryExecutionOpts<TVars> = {
   cachePolicy: CachePolicy;
   variables: TVars;
-}
+};
```

Comment:
Thanks, just a tiny nitpick. 

Do you mind keeping it `export interface` instead of `export type? just a preference of mine. Also, don't you need this type exported in `index.ts` or is that not necessary?

Reply:
Ah I see, that's why the interface still wasn't exported in the final villus.d.ts file. If i export it from the index.ts file it is also fine. Changed it back to an interface :+1: